apps/aeutils/src/aeu_env.erl

```diff
@@ -286,18 +297,30 @@ config_file() ->
             F
     end.
 
-data_dir(Name) when is_atom(Name) ->
-  filename:join([setup:data_dir(), Name]).
+deprecated_config_file() ->
+    case os:getenv("EPOCH_CONFIG") of
+        false ->
+            search_deprecated_config();
+        F ->
+            F
+    end.
 
 search_default_config() ->
-    Dirs = [filename:join([os:getenv("HOME"), ".epoch", nodename()]),
+    Dirs = [filename:join([os:getenv("HOME"), ".aeternity", "aeternity"]),
             setup:home()],
+    search_config(Dirs, "aeternity.{json,yaml}").
+
+search_deprecated_config() ->
+    Dirs = [filename:join([os:getenv("HOME"), ".epoch", "epoch"]),
```

Comment:
possible BC break, but IIRC we agreed it's OK - very low probability

Reply:
actually not possible until someone manipulated with `vm.args` by hand...